apps/pie-storybook/stories/pie-assistive-text.stories.ts

```diff
@@ -2,35 +2,55 @@ import { html } from 'lit';
 
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-assistive-text';
-import { AssistiveTextProps } from '@justeattakeaway/pie-assistive-text';
+import { AssistiveTextProps, variants } from '@justeattakeaway/pie-assistive-text';
 /* eslint-enable import/no-duplicates */
 
 import { type StoryMeta } from '../types';
 import { createStory } from '../utilities';
 
 type AssistiveTextStoryMeta = StoryMeta<AssistiveTextProps>;
 
-const defaultArgs: AssistiveTextProps = {};
+const defaultArgs: AssistiveTextProps = {
+    variant: 'default',
+    slot: 'Assistive Text',
+};
 
 const assistiveTextStoryMeta: AssistiveTextStoryMeta = {
     title: 'Assistive Text',
     component: 'pie-assistive-text',
-    argTypes: {},
+    argTypes: {
+        variant: {
+            description: 'Set the variant of the assistive text.',
+            control: 'select',
+            options: variants,
+            defaultValue: {
+                summary: 'default',
+            },
+        },
+    },
```

Comment:
I think you will need to add arg type for the slot as well so users can type what they want. [here](https://github.com/justeattakeaway/pie/pull/1248/files#diff-35c71a2a7b57dcd5e06095f70907a848a84003a2900089d86b7ca38695bd0687R73) is an exmaple. you will also need to update the template a bit

Reply:
I don't know if that's needed actually, I'm able to type anything in the slot on the deployed storybook already 🎉 
<img width="1481" alt="Screenshot 2024-02-07 at 08 37 35" src="https://github.com/justeattakeaway/pie/assets/49618712/d5186042-326b-4cec-81de-b48981545934">

Reply:
oh cool, then the main reason we need it, so we can [sanitize](https://github.com/justeattakeaway/pie/pull/1248/files#diff-35c71a2a7b57dcd5e06095f70907a848a84003a2900089d86b7ca38695bd0687R105) the slot value passed by the user.

Reply:
good spot @raoufswe!

Reply:
ahh okay, I copied this from tag so will change there too in this PR! 👍